tasks/upload_finisher.py

```diff
@@ -401,6 +402,10 @@ def finish_reports_processing(
             commit.state = "skipped"
 
         if self.should_clean_labels_index(commit_yaml, processing_results):
+            # NOTE: according to tracing, the cleanup task is never actually run.
+            # reasons for that might be that we indeed *never* have any flags
+            # configured for `carryforward_mode=labels`, or the logic is somehow wrong?
```

Comment:
I think it _might_ be because no repos use the label index at all
(see [here](https://github.com/codecov/worker/blob/ad3621cbee3619ec1b47ee55a1050e8be4343dda/services/report/raw_upload_processor.py#L87))

Reply:
well yes, though the conditional above in `UploadFinisher` is not looking at that feature flag at all.